core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -642,7 +642,7 @@ public Task getPendingTaskByWorkflow(String taskReferenceName, String workflowId
      */
     @VisibleForTesting
     void completeWorkflow(Workflow wf) {
-        LOGGER.debug("Completing workflow execution for {}", wf.getWorkflowId());
+        LOGGER.info("Completing workflow execution for {}", wf.getWorkflowId());
```

Comment:
you see value in this log and the log below?

Reply:
Changed to debug